app/subscriber/src/features/my-reports/edit/content/ContentEditForm.tsx

```diff
@@ -27,17 +30,67 @@ export interface IContentEditFormProps {
 export const ContentEditForm = React.forwardRef<HTMLDivElement | null, IContentEditFormProps>(
   ({ disabled }, ref) => {
     const [{ userInfo }] = useApp();
+    const [{ myTonePool, init }, { storeMyTonePool }] = useProfileStore();
     const [, { updateReport }] = useReports();
     const [, { addContent, updateContent }] = useContent();
+    const [, { addMyTonePool, getMyTonePool }] = useTonePool();
     const { values, onNavigate, isSubmitting, setSubmitting, setValues, activeRow, setActiveRow } =
       useReportEditContext();
 
     const [form, setForm] = React.useState<IReportInstanceContentForm | undefined>(activeRow);
     const [errors, setErrors] = React.useState<IContentValidationErrors>();
+    const [userFormTonePool, setUserFormTonePool] = React.useState<
+      IContentTonePoolModel | undefined
+    >(undefined);
 
     const instance = values.instances.length ? values.instances[0] : undefined;
     const userId = userInfo?.id ?? 0;
 
+    React.useEffect(() => {
+      const getTonePool = async () => {
+        try {
+          if (!init.myTonePool && userId !== 0) {
+            const response = await getMyTonePool(userId);
+            if (response?.id) {
+              storeMyTonePool(response);
+            } else {
+              // If no valid tone pool exists, proceed to create one
+              await createTonePool(userId);
```

Comment:
This will result in creating a tonepool for each user that edits content in a report.  Many users will never want to use their own tonepool.  They only need to be created if they choose to change the sentiment.

Reply:
So the creation of the tonePool should be until they click on the sentiment icon at the bottom.